ui/Filters.elm

```diff
@@ -0,0 +1,1030 @@
+    , "47.0b9"
+    , "48.0.1"
+    , "48.0.2"
+    , "48.0"
+    , "48.0b1"
+    , "48.0b10"
+    , "48.0b2"
+    , "48.0b3"
+    , "48.0b4"
+    , "48.0b5"
+    , "48.0b6"
+    , "48.0b7"
+    , "48.0b9"
+    , "49.0.1-funnelcake87"
+    , "49.0.1-funnelcake88"
+    , "49.0.1-funnelcake89"
+    , "49.0.1-funnelcake90"
+    , "49.0.1"
+    , "49.0.2"
+    , "49.0"
+    , "49.0b1"
+    , "49.0b10"
+    , "49.0b2"
+    , "49.0b3"
+    , "49.0b4"
+    , "49.0b5"
+    , "49.0b6"
+    , "49.0b7"
+    , "49.0b8"
+    , "49.0b9"
+    , "5.0.1"
+    , "5.0"
+    , "5.0b1"
+    , "5.0b2"
+    , "5.0b3"
+    , "5.0b5"
+    , "5.0b6"
+    , "5.0b7"
+    , "50.0-funnelcake93"
+    , "50.0-funnelcake94"
+    , "50.0.1-funnelcake91"
+    , "50.0.1-funnelcake92"
+    , "50.0.1-funnelcake93"
+    , "50.0.1-funnelcake94"
+    , "50.0.1"
+    , "50.0.2-funnelcake91"
+    , "50.0.2-funnelcake92"
+    , "50.0.2-funnelcake93"
+    , "50.0.2-funnelcake94"
+    , "50.0.2"
+    , "50.0"
+    , "50.0b1"
+    , "50.0b10"
+    , "50.0b11"
+    , "50.0b2"
+    , "50.0b3"
+    , "50.0b4"
+    , "50.0b5"
+    , "50.0b6"
+    , "50.0b7"
+    , "50.0b8"
+    , "50.0b9"
+    , "50.1.0-funnelcake95"
+    , "50.1.0-funnelcake96"
+    , "50.1.0-funnelcake97"
+    , "50.1.0"
+    , "51.0.1-funnelcake100"
+    , "51.0.1-funnelcake101"
+    , "51.0.1-funnelcake102"
+    , "51.0.1-funnelcake103"
+    , "51.0.1-funnelcake104"
+    , "51.0.1-funnelcake98"
+    , "51.0.1-funnelcake99"
+    , "51.0.1"
+    , "51.0"
+    , "51.0b1"
+    , "51.0b10"
+    , "51.0b11"
+    , "51.0b12"
+    , "51.0b13"
+    , "51.0b14"
+    , "51.0b2"
+    , "51.0b3"
+    , "51.0b4"
+    , "51.0b5"
+    , "51.0b6"
+    , "51.0b7"
+    , "51.0b8"
+    , "51.0b9"
+    , "52.0-funnelcake101"
+    , "52.0-funnelcake102"
+    , "52.0-funnelcake103"
+    , "52.0-funnelcake104"
+    , "52.0.1-funnelcake101"
+    , "52.0.1-funnelcake102"
+    , "52.0.1-funnelcake103"
+    , "52.0.1-funnelcake104"
+    , "52.0.1"
+    , "52.0.1esr"
+    , "52.0.2-funnelcake107"
+    , "52.0.2-funnelcake108"
+    , "52.0.2-funnelcake109"
+    , "52.0.2"
+    , "52.0.2esr"
+    , "52.0"
+    , "52.0b1"
+    , "52.0b2"
+    , "52.0b3"
+    , "52.0b4"
+    , "52.0b5"
+    , "52.0b6"
+    , "52.0b7"
+    , "52.0b8"
+    , "52.0b9"
+    , "52.0esr"
+    , "52.1.0esr"
+    , "52.1.1esr"
+    , "52.1.2esr"
+    , "53.0-funnelcake110"
+    , "53.0-funnelcake111"
+    , "53.0-funnelcake112"
+    , "53.0-funnelcake113"
+    , "53.0.2-funnelcake110"
+    , "53.0.2-funnelcake111"
+    , "53.0.2-funnelcake112"
+    , "53.0.2-funnelcake113"
+    , "53.0.2"
+    , "53.0.3"
+    , "53.0"
+    , "53.0b1"
+    , "53.0b10"
+    , "53.0b2"
+    , "53.0b3"
+    , "53.0b4"
+    , "53.0b5"
+    , "53.0b6"
+    , "53.0b7"
+    , "53.0b8"
+    , "53.0b9"
+    , "54.0b1"
+    , "54.0b10"
+    , "54.0b11"
+    , "54.0b12"
+    , "54.0b2"
+    , "54.0b3"
+    , "54.0b4"
+    , "54.0b5"
+    , "54.0b6"
+    , "54.0b7"
+    , "54.0b8"
+    , "54.0b9"
```

Comment:
I think we can start to add 55 and 56 releases as well so that we will have the filter already for the All hands demo.